src/arch/xtensa/configs/cannonlake_defconfig

```diff
@@ -2,3 +2,6 @@ CONFIG_SMP=y
 CONFIG_CANNONLAKE=y
 CONFIG_CAVS_DMIC=y
 CONFIG_CAVS_SSP=y
+CONFIG_CORE_COUNT=4
+CONFIG_LP_MEMORY_BANKS=1
+CONFIG_HP_MEMORY_BANKS=47
```

Comment:
can we add a boilerplate CONFIG_CAVS_SOUNDWIRE while we are at it, starting with CNL?

Reply:
@plbossart Why? Isn't it better to introduce config when it's going to be used?

Reply:
@plbossart I wouldn't do SDW here since it has nothing to do with core/memory pr. I am already told that i should split it ;)